src/metldata/schemapack_/transform/base.py

```diff
@@ -111,7 +119,7 @@ class TransformationDefinition(Generic[Config]):
         ...,
         description=(
             "A function that checks the assumptions made about the input model."
-            " Raises a ModelAssumptionError if the assumptions are not met."
+            "Raises a ModelAssumptionError if the assumptions are not met."
```

Comment:
```suggestion
            " Raises a ModelAssumptionError if the assumptions are not met."
```

Reply:
Is there a reason why this blank char always is removed again?

Reply:
Not sure. Maybe some formatting issue. It also is missing the plusses. Is fixed in latest commit.